packages/api/src/beacon/routes/lightclient.ts

```diff
@@ -88,116 +84,64 @@ export type Endpoints = {
+            for (const [i, update] of data.entries()) {
+              json.push(getLightClientForkTypes(meta.versions[i]).LightClientUpdate.toJson(update));
             }
             return json;
           },
           fromJson: (data, meta) => {
+            const updates = data as unknown[];
             const value: allForks.LightClientUpdate[] = [];
-            for (let i = 0; i < meta.versions.length; i++) {
+            for (let i = 0; i < updates.length; i++) {
               const version = meta.versions[i];
-              value.push(getLightClientForkTypes(version).LightClientUpdate.fromJson((data as unknown[])[i]));
-            }
-            return value;
-          },
-          serialize: (data, meta) => {
-            if (data.length !== meta.versions.length) {
-              throw new Error("TODO"); // TODO
-            }
-            if (!beaconConfig) {
-              throw new Error("TODO"); // TODO
-            }
-            const bufs: Uint8Array[] = [];
-            for (const [i, version] of meta.versions.entries()) {
-              const forkDigest = beaconConfig.forkName2ForkDigest(version);
-              const serialized = getLightClientForkTypes(version).LightClientUpdate.serialize(
-                data[i] as deneb.LightClientUpdate
-              );
-              const length = ssz.UintNum64.serialize(4 + serialized.length);
-              bufs.push(length, forkDigest, serialized);
-            }
-            return Buffer.concat(bufs);
-          },
-          deserialize: (data) => {
-            if (!beaconConfig) {
-              throw new Error("TODO"); // TODO
-            }
-            let offset = 0;
-            const value: allForks.LightClientUpdate[] = [];
-            while (offset < data.length) {
-              const length = ssz.UintNum64.deserialize(data.subarray(offset, offset + 8));
-              const forkDigest = ssz.ForkDigest.deserialize(data.subarray(offset + 8, offset + 12));
-              const version = beaconConfig.forkDigest2ForkName(forkDigest);
-              value.push(
-                getLightClientForkTypes(version).LightClientUpdate.deserialize(
-                  data.subarray(offset + 12, offset + length)
-                )
-              );
-              offset += length;
+              value.push(getLightClientForkTypes(version).LightClientUpdate.fromJson(updates[i]));
             }
             return value;
           },
         },
         meta: {
           toJson: (meta) => meta,
           fromJson: (val) => val as {versions: ForkName[]},
-          toHeadersObject: (meta) => ({
-            [MetaHeader.Version]: meta.versions.join(","),
-          }),
-          fromHeaders: (headers) => ({
-            // TODO: this header format is not spec compliant, do we care?
-            versions: headers.getRequired(MetaHeader.Version).split(",") as ForkName[],
```

Comment:
as spec already has a well-defined ssz response, we should make sure to be compliant, also ensure we work with Nimbus